src/execution/expression_executor/execute_function.cpp

```diff
@@ -76,7 +76,21 @@ void ExpressionExecutor::Execute(const BoundFunctionExpression &expr, Expression
 	arguments.Verify();
 
 	D_ASSERT(expr.function.function);
+	// #ifdef DEBUG
```

Comment:
eventually this will be added back in. For now it will disrupt to much of the developer work flow.